src/test/java/tests/SearchFieldTest.java

```diff
@@ -18,15 +18,17 @@ private void initTest() {
     @Test(description = "SPRIN-64: Clear the search field")
     public void testClearSearchField() {
 
-        softAssert.assertTrue(headerFragment.getButtonClearSearchFieldPresence().isDisplayed(), "sad");
-        softAssert.assertTrue(!headerFragment.isButtonDisplayed(), "The button is visible");
+        softAssert.assertTrue(!headerFragment.getClearSearchFieldButton().isDisplayed(),
```

Comment:
use assertFalse for this it will be clearer for understanding:
```suggestion
        softAssert.assertFlase(headerFragment.getClearSearchFieldButton().isDisplayed(),
```